### PR TITLE
Fix install linux dep priviledge

### PR DESCRIPTION
**Pull request by congqixia**
See also #19806 #19920

/kind improvement

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

File: scripts/install_deps.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ function install_linux_deps() {
     exit 1
   fi
   # install cmake
-  wget -qO- "https://cmake.org/files/v3.24/cmake-3.24.0-linux-x86_64.tar.gz" | tar --strip-components=1 -xz -C /usr/local
+  wget -qO- "https://cmake.org/files/v3.24/cmake-3.24.0-linux-x86_64.tar.gz" | sudo tar --strip-components=1 -xz -C /usr/local
```
Comment (matrixji) [inline scripts/install_deps.sh:48]: Suggest using pip to install cmake, which not requires privileged

Comment (yhmo) [reply scripts/install_deps.sh:48]: @matrixji 
Please create a pr later to use pip to install cmake.

**Pull request merged**